tachyon/zk/base/commitments/gwc_extension.h

```diff
@@ -16,6 +16,10 @@
 #include "tachyon/crypto/commitments/kzg/gwc.h"
 #include "tachyon/zk/base/commitments/univariate_polynomial_commitment_scheme_extension.h"
 
+#if TACHYON_CUDA
+#include "tachyon/device/gpu/gpu_memory.h"
+#endif
```

Comment:
I think this is redundant. Since `gwc_.kzg().d_g1_powers_of_tau()` returns `device::gpu::GpuMemory<G1Point>`, it's safe to assume that the necessary header is already included with the implementation of kzg.